server/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Well done for adding this file. I think you committed your node_modules folder before you added this .gitignore, which is unnecessary and will make the project much bigger to download. You can remove node_modules from the git tracking by running this command in the terminal `git rm -r --cached node_modules`. Then next time you commit & push they will be removed from github, but not your local machine